modules/models/src/main/java/org/eclipse/xpanse/modules/models/billing/ServicePrice.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.billing;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.eclipse.xpanse.modules.models.billing.enums.Currency;
+import org.eclipse.xpanse.modules.models.billing.enums.PricingPeriod;
+
+/**
+ * Defines the price of the managed service.
+ */
+@Data
+public class ServicePrice {
```

Comment:
Another point to be considered here is -> if service pricing is pay_per_use but if license fees or markup price is "one-time" then we must show that separately. 

So better to add two different prices to this model -> RecurringPrice, OneTimePaymentPrice

Reply:
What `one-time` means?

Reply:
one-time is the price what the customer pays just once. So even if he uses the service for many years or months. still he pays it only once. 
Yes, this is not available in the Huaweicloud APIs at the moment. Mostly we need it for other CSPs or also for HuaweiCloud in future. 
in any case, this amount will always be taken from the OCL directly.

Reply:
Understood and updated code.